packages/client-sdk-nodejs/src/config/configuration.ts

```diff
@@ -284,4 +289,8 @@ export class CacheConfiguration implements Configuration {
       compression: compressionStrategy,
     });
   }
+
+  hasCompressionStrategy(): boolean {
+    return this.compression !== undefined;
+  }
```

Comment:
We can use this in the Redis wrappers to decide whether they should compress or not.